### DISCUSSION TITLE
Mac How to send £ character?

### DISCUSSION THREAD

**Discussion by benfrain** [kovidgoyal/kitty#5056] [General]
On my keyboard, outside of Kitty I can use <kbd>Option+3</kbd> to get a `£` symbol. Inside Kitty that does nothing.

I have `macos_option_as_alt yes` set in my conf.

How do I allow Kitty to recieve the relevant keycodes I would typically get when using <kbd>Option</kbd>?

**Comment by kovidgoyal** [comment]
You want

macos_option_as_alt no



On Wed, May 04, 2022 at 07:28:12AM -0700, Ben Frain wrote:
> On my keyboard, outside of Kitty I can use <kbd>Option+3</kbd> to get a `£` symbol. Inside Kitty that does nothing.
> 
> I have `macos_option_as_alt yes` set in my conf.
> 
> How do I allow Kitty to recieve the relevant keycodes I would typically get when using <kbd>Option</kbd>? 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5056
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Comment by benfrain** [comment]
Ah, that simple, hey? Which makes me worry that I have it set to `yes` so it lets me use it in neovim?? Hmmmm

**Reply by kovidgoyal** [reply]
Read the docs for that option you can have the left and right option keys work differently

**Reply by benfrain** [reply]
thanks, got it set to `left` now so I can use it as a alt in Neovim and use the right option when I need to get to unicodes. Wonderful, thanks
